variety_lib/varietyconfig.py

```diff
@@ -23,7 +23,7 @@
     # for easy development / running from source.
     __variety_data_directory__ = "../data"
 __license__ = "GPL-3"
-__version__ = "0.8.12"
+__version__ = "0.8.13"
```

Comment:
The app version should not be changed in a PR. This needs to be done separately (with corresponding changelog updates) and tagged by a maintainer

Reply:
Okey doke. Thanks for the feedback